app/plugins/Files/js/helpers.js

```diff
@@ -3,7 +3,7 @@
 // Library for communicating with Sia-UI
 const IPC = require('ipc');
 // Library for arbitrary precision in numbers
-const BigNumber = require('bignumber.js');
+const BigNumber = require('../../js/bignumber.min.js');
```

Comment:
I will confess it's not the best plan in the world.  I still haven't fully embraced the node way yet.

Things will continue to evolve, but I needed to be able to build the app in the meantime.